azkaban-common/src/main/java/azkaban/executor/ExecutorApiGateway.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2015 LinkedIn Corp.
```

Comment:
2017

Reply:
FYI I set up the copyright notice in intellij and it will auto update.
I plan to share how I did it in a wiki when I find time to do so.